### ISSUE TITLE
[a11y] Cannot open saved queries

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1186]
**Describe the bug**
When attempting to open a saved query (*except for Collection Object*), this error occurs.

```
Unexpected Error

An unexpected error has occurred
Please reload the page and try again.
If this issue persists, please contact your IT support or if this is a Specify Cloud database, contact [support@specifysoftware.org](mailto:support@specifysoftware.org)

Error Message
TypeError: Cannot read properties of undefined (reading 'type')

    at w (https://a11y.test.specifysystems.org/static/js/636.3db06bff2c62c14dfb33.bundle.js:2:78648)
    at ul
    at Ul
    at z (https://a11y.test.specifysystems.org/static/js/693.0e9823db0dbd8b92a479.bundle.js:2:21451)
    at div
    at div
    at form
    at Form
    at section
    at Section
    at X (https://a11y.test.specifysystems.org/static/js/693.0e9823db0dbd8b92a479.bundle.js:2:22519)
    at te (https://a11y.test.specifysystems.org/static/js/693.0e9823db0dbd8b92a479.bundle.js:2:27665)
    at https://a11y.test.specifysystems.org/static/js/693.0e9823db0dbd8b92a479.bundle.js:2:27970
    at c (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:51405)
    at E (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:26468)
```

**Comment by maxxxxxdlp** [comment]
Fixed
